TACC/Core-Portal
# Bug/FP-1110: Shared workspace description character limit

## Overview: ##
Added character limit to workspace description (800 chars) and title (70 chars). Also handled overflow and wrap for shared workspace title and breadcrumbs.

## Related Jira tickets: ##

* [FP-1110](https://jira.tacc.utexas.edu/browse/FP-1110)

## Summary of Changes: ##

## Testing Steps: ##
1. Try adding workspace with title over 70 characters and ensure that it doesn't allow you.
2. Try repeating the above step in the edit description (800 characters) modal.
3. Make sure that the title text in breadcrumbs and the section header wrap and handle overflow nicely.

## UI Photos:
![peek](https://user-images.githubusercontent.com/9425579/125138922-26e30480-e0d5-11eb-8f92-a3c4bf7f6ff2.gif)
![Selection_001](https://user-images.githubusercontent.com/9425579/125138926-28acc800-e0d5-11eb-8d32-05b1bfd642e7.png)
![Selection_002](https://user-images.githubusercontent.com/9425579/125138928-29ddf500-e0d5-11eb-86ef-7324a602890b.png)


## Notes: ##
Wasn't sure if the best way to handle the overflow was manually adding media queries.

Also, since we allow whitespace in descriptions as it is, the character limit applies to whitespace, wasn't sure if I should've trimmed characters before testing limit.
